test/blackbox-tests/test-cases/melange/public.t/run.t

```diff
@@ -0,0 +1,30 @@
+Test the public libs flow when using `melange.emit` stanza
+
+Cmj rules should include --bs-package-output
+  $ dune rules my_project/app/.app.objs/melange/app.cmj | 
+  > grep -e "--bs-package-output" --after-context=1 
+      --bs-package-output
+      _build/default/my_project/app
```

Comment:
@anmonteiro I am using the build path for `--bs-package-output`. I saw it does not seem to matter, but unsure if it's safe. Could you confirm please?